<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>del_setting_dialog</class>
 <widget class="QDialog" name="del_setting_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="del_only_line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>删除1项</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="del_selected_line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中项</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="del_Allline">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>删除所有</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remember_cheBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住我的选择，不再提醒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>141</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确定(O)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>141</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消(C)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>141</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>应用(A)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
